storybook/mocks/DotherSide/qmldir

```diff
@@ -1 +1,2 @@
 StatusSyntaxHighlighter 0.1 StatusSyntaxHighlighter.qml
+RXValidator 0.1 RXValidator.qml
```

Comment:
I would opt for waiting a bit and merging this PR after https://github.com/status-im/status-desktop/pull/9457. Then it will be no need to create mocks - it'll be possible to use those types directly, with their full functionality.

Reply:
OK, let's see who wins the merge race :)

Reply:
Removed it here